blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/ChangeSetResource.java

```diff
@@ -107,6 +107,6 @@ public BlueChangeSetEntry setCheckoutCount(int checkoutCount) {
 
     @Override
     public Link getLink() {
-        return parent.getLink().rel("changeset/"+getCommitId());
+        return parent.getLink().rel(getCommitId());
```

Comment:
Shouldn't this just be `parent.getLink().rel("changeSet")`?

Reply:
The container is "changeSet", this is the Id of the actual element inside of the container.
so parent.getlink() === "changeSet"
...
I think at least.

Reply:
Right, I see. Poorly named classes (not your doing I know) confused me :(